src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -11,16 +11,14 @@ export default class DatagridStore {
     disposer: () => void;
     resourceKey: string;
     options: Object;
-    observableOptions: Object;
+    observableOptions: {
```

Comment:
Don't you think it would be a good idea to put that into `types.js`? Then you could also reuse it in the constructor as well, leading to better error messages.